introduction.md

```diff
@@ -1 +1,5 @@
-My name is Sai Teja. My fav path on codeschool is javascript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?